app/nagbot.py

```diff
@@ -37,8 +37,7 @@ class Nagbot(object):
     @staticmethod
     def notify_internal(channel, dryrun):
         pricing = PricingData()
-        instances = sqaws.list_ec2_instances(pricing)
```

Comment:
Should these two `list_` methods be removed now?

Reply:
yep, getting rid of them now - just wanted to be sure the new one looked good before deleting!